packages/protocol/contracts/layer2/based/TaikoL2.sol

```diff
@@ -131,11 +134,13 @@ contract TaikoL2 is EssentialContract, IBlockHash, TaikoL2Deprecated {
     /// transaction, and any subsequent calls will revert with L2_PUBLIC_INPUT_HASH_MISMATCH.
     /// @param _anchorBlockId The `anchorBlockId` value in this block's metadata.
     /// @param _anchorStateRoot The state root for the L1 block with id equals `_anchorBlockId`.
+    /// @param _anchorInput An arbitrary bytes32 input chosen by the block proposer.
     /// @param _parentGasUsed The gas used in the parent block.
     /// @param _baseFeeConfig The base fee configuration.
     function anchorV2(
```

Comment:
Shall we rename this `anchorV3`? since its a break change in params.

Reply:
renamed anchorV2 to anchorV3